File: web/app/FlightSeatPrice.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class FlightSeatPrice extends Model
{
    protected $fillable = [
      'price',
      'flight_id',
      'aircraft_id',
      'flight_seat_id'
    ];
    public function flight(){
      return $this->hasOne('App\Flight', 'id', 'flight_seat_id');
    }

    public function flight_seat(){
      return $this->hasOne('App\FlightSeat', 'id', 'flight_seat_id');
    }
}

File: android/app/src/main/java/tut/flightbookingsystem/model/FlightSeatPrice.java
package tut.flightbookingsystem.model;

import java.io.Serializable;

public class FlightSeatPrice implements Serializable {
    public long id;
    public double price;
    public long flight_id;
    public long aircraft_id;
    public long flight_seat_id;
    public FlightSeat flight_seat;
    public String created_at;
    public String updated_at;
}
